frontend/src/Metamaps/PasteInput.js

```diff
@@ -70,11 +67,23 @@ const PasteInput = {
       Import.handleURL(text, coords)
     } else if (text[0] === '{') {
       Import.handleJSON(text)
-    } else if (text.match(/\t/)) {
+    } else if (text.match(/[Tt]opics\t/) || text.match(/[Ss]ynapses\t/)) {
```

Comment:
I made this change on the branch - can you test it when you're back developing and make sure it's not screwing anything up